docs/feature-store/sources-targets.md

```diff
@@ -0,0 +1,47 @@
+(sources-targets)=
+# Sources and targets
+
+
+- [Sources](#sources)
+- [Targets](#targets)
+- [Partitioning on Parquet target](#partitioning-on-parquet-target)
+
+
+
+## Sources
+| Class name                                                                                                        | Description                                                                     | storey | spark | pandas |batch or event|
+| --------------------------------------------------                                                                | ---------------------------------                                               | ---    | ---   | ---    |---|
+| [mlrun.datastore.BigQuerySource](../api/mlrun.datastore.html#mlrun.datastore.BigQuerySource)                      | Reads Google BigQuery query results as input source for a flow.                 | N      | Y     | Y      |batch|
```

Comment:
since batch-or-event is actually an attribute of the source, I think it's misleading to provide this info alongside columns that convey whether a particular ingestion engine support it.
Instead, I would add it to the Description:
`Reads Google BigQuery query results as input source for a flow ("Batch" source)`
`Sets the kafka source for the flow (Event-based source)`